staff/carlos-calvo/cv/index.html

```diff
@@ -0,0 +1,180 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>CV Carlos Calvo</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" type="text/css" media="screen" href="main.css" />
+    <script src="main.js"></script>
+</head>
+<body>
+    <header>CV CARLOS CALVO LOPEZ</header>
+    <img src=""></img>
+    <hr/>
+    <section>
+        <h1><b>PERSONAL INFORMATION</b></h1>
+        <ul>
+            <li>Name: Carlos</li>
+            <li>Surname: Calvo</li>
+            <li>Birth Date: 09/12/1986</li>
+            <li>mail: carloscl1986@gmail.com</li>
+            <li>Mobile Phone: 620092996</li>
+        </ul>
+    </section>
+    <hr/>
+    <section>
+        <h1><b>FORMATION</b></h1>
```

Comment:
You can add every h1 from every section inside a header element

Reply:
OK! Nice :)

Reply:
Sorry Carlos, I've asked to Manu and he told me that when you have only one heading you can remove the `<heading>` element.
Sorry for the bad correction